.gitignore

```diff
@@ -2,8 +2,9 @@
 ## files generated by popular Visual Studio add-ons.
 
 # Tool Runtime Dir
-/vNext/src/[Tt]ools/
+/vNext/[Tt]ools/
 /vNext/src/[Bb]ootstrap/
```

Comment:
I am not sure. We will need to clean this part up when merging the 2 trees I think.